sasl_bind.c

```diff
@@ -0,0 +1,21 @@
+#include <ldap.h>
+#include "sasl_bind.h"
+
+static int
+sasl_trivial_interact (LDAP * ld, unsigned flags, void *defaults,
+                       void *sasl_interact)
+{
+  (void) ld;
+  (void) flags;
+  (void) defaults;
+  (void) sasl_interact;
+  return LDAP_SUCCESS;
+}
+
+int
+trivial_external_sasl_bind (LDAP * ld)
+{
+  return ldap_sasl_interactive_bind_s (ld, NULL, "EXTERNAL", NULL, NULL,
```

Comment:
I asked about the _authorization_ identity, which can be different from the authentication identity. See https://tools.ietf.org/html/rfc2222#section-7.4 and http://www.openldap.org/doc/admin24/sasl.html#SASL%20Proxy%20Authorization.